src/dpg_slam/parameters.h

```diff
@@ -49,37 +49,48 @@ namespace dpg_slam {
         /**
          * Number of sectors per node.
          */
-        uint8_t num_sectors_;
+        uint8_t num_sectors_ = 10;
```

Comment:
I'd either just leave this to use the default value or delete the default value.

Reply:
fixed.